connection.schema.json

```diff
@@ -49,6 +49,11 @@
     "workload": {
       "title": "Workload",
       "type": "string"
+    },
+    "showSystemObj": {
```

Comment:
It should default to false because the normal SQLTools drivers do not include system objects.  

I agree it should be plural.   How about showSystemTables?  showSystemObjects?  I find showSystemObjs to be hard to parse.   I prefer showSystemTables.  I realize this includes the tables and the schemas they are in, but I think that is clear.

Reply:
It's also for views in some cases though, so it's schemas, views, and tables. Objects might object-ively be a better option😄

Reply:
Then how about showSystemObjects?   Screens are big these days and there's less need to abbreviate...